modules/tinymce/src/plugins/lists/main/ts/ui/Dialog.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import Editor from 'tinymce/core/api/Editor';
+import { isOlNode } from '../core/NodeType';
+import { getParentList } from '../core/Selection';
+
+const open = (editor: Editor) => {
+  const dom = editor.dom;
+
+  // Find the current list and skip opening if the selection isn't in an ordered list
+  const parentList = getParentList(editor);
+  if (!isOlNode(parentList)) {
+    return;
+  }
+
+  editor.windowManager.open({
+    title: 'List Properties',
+    body: {
+      type: 'panel',
+      items: [
+        {
+          type: 'input',
+          name: 'start',
+          label: 'Start list at number',
+          inputMode: 'numeric'
+        }
+      ]
+    },
+    initialData: {
+      start: dom.getAttrib(parentList, 'start') || '1'
+    },
+    buttons: [
+      {
+        type: 'cancel',
+        name: 'cancel',
+        text: 'Cancel'
+      },
+      {
+        type: 'submit',
+        name: 'save',
+        text: 'Save',
+        primary: true
+      }
+    ],
+    onSubmit: (api) => {
+      const data = api.getData();
+      editor.undoManager.transact(() => {
+        dom.setAttrib(parentList, 'start', data.start === '1' ? '' : data.start);
```

Comment:
`parentList` won't change while the dialog is open? I guess it's unlikely without RTC.

Reply:
Good point, it shouldn't ever do that, but to be on the safe side I'll update it to re-fetch the parent list here. :+1: